wgpu-hal/src/gles/device.rs

```diff
@@ -308,6 +331,7 @@ impl super::Device {
         Ok(program)
     }
 
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Otherwise getting
```
error: this function has too many arguments (8/7)
```
https://github.com/gfx-rs/wgpu/actions/runs/4083942757/jobs/7147934141